index.js

```diff
@@ -313,31 +313,16 @@ module.exports = {
       return pull(
         pull.values([0]),
         pull.asyncMap((n, cb) => {
-          get(groupId, (err, group) => {
+          getPickedEpoch(groupId, (err, pickedEpoch) => {
             // prettier-ignore
-            if (err) return cb(clarify(err, 'Failed to get group info when listing members'))
+            if (err) return cb(clarify(err, 'todo'))
 
-            if (group.excluded) {
-              return cb(
-                new Error("We're excluded from this group, can't list members")
-              )
-            } else {
-              const source = pull(
-                ssb.db.query(
-                  where(and(isDecrypted('box2'), type('group/add-member'))),
-                  opts.live ? live({ old: true }) : null,
```

Comment:
Realized that we'll need to support `opts.live` (or?) which makes this pr trickier. But I think it should be possible without too much work, box2/keyring will have to have a live `get` when changes happen, e.g. new picked write key, and getMembers will have to support live as well, but that doesn't look that hard either.